src/app/clusters/ota-requestor/GenericOTARequestorDriver.cpp

```diff
@@ -164,9 +170,6 @@ void GenericOTARequestorDriver::UpdateNotFound(UpdateNotFoundReason reason, Syst
         }
         break;
     }
```

Comment:
Why remove the default clause?

Reply:
I removed it to force compile-time errors if a new enumeration value is added, but not handled here (i.e. the dev forgetting to, for instance). :shrug: Might be useful?